Phonebook.Frontend/src/app/shared/pipes/telephone.pipe.ts

```diff
@@ -5,6 +5,6 @@ import { Pipe, PipeTransform } from '@angular/core';
 })
 export class TelephonePipe implements PipeTransform {
   public transform(telephoneNumber: string): string {
```

Comment:
Auch hier bitte die Funktion und Ausgangswerte im Model mit anpassen! 
```suggestion
  public transform(telephoneNumber: null | string): string {
```